app/scripts/services/projects.js

```diff
@@ -32,10 +32,13 @@ angular.module('openshiftConsole')
                       return DataService
                               .get('projects', projectName, context, {errorNotification: false})
                               .then(function(project) {
```

Comment:
Thinking about any use of the `rules` in the controller, but cant think of any, since the use of the `canI` filter. Any ideas ?